src/hooks/sync-sites/use-sync-pull.ts

```diff
@@ -90,6 +91,9 @@ export function useSyncPull( {
 			if ( ! client ) {
 				return;
 			}
+
+			console.groupCollapsed( 'Sync Pull' );
```

Comment:
I'm not entirely convinced we need that debugging in the browser console, but thanks to the collapsed message group feature and clear messages, the whole chain looks okay.

Reply:
Sure, we can work on a toggle in the future. For now, we should be good as soon as they go to the browser console, as the built binary shouldn't include the dev console at all.